openy.install

```diff
@@ -60,3 +60,29 @@ function openy_update_8003() {
   // Enable module.
   \Drupal::service('module_installer')->install(['config_import']);
 }
+
+/**
+ * Enable Data layer modules & revert needed configuration.
+ */
+function openy_update_8004() {
+  // Enable module.
+  \Drupal::service('module_installer')->install(
+    ['libraries', 'datalayer', 'openy_datalayer']
+  );
+
+  // Feature revert updated features.
+  $modules = [
+    'openy_loc_branch',
+    'openy_loc_camp',
+    'openy_loc_facility',
+    'openy_node_blog',
+    'openy_node_category',
+    'openy_node_mbrshp',
+    'openy_node_program',
+  ];
+  $module_handler = \Drupal::service('module_handler');
+  $modules = array_filter($modules, function ($module) use ($module_handler) {
+    return $module_handler->moduleExists($module);
+  });
+  \Drupal::service('features.manager')->import($modules);
```

Comment:
We shouldn't revert whole feature. There are diffrerent levels that can be used in upgrade path:
- Revert feature
- Revert some specific file
- Revert some specific property in config file

In upgrade path for the distribution we should use only 3rd, https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md#revert-only-specific-property-from-config.